indra/newview/VIEWER_VERSION.txt

```diff
@@ -1 +1 @@
-7.1.2
\ No newline at end of file
+7.1.2
```

Comment:
This was failing the pre-commit hook checking for an EOF, so I opened and re-saved it